### ISSUE TITLE
[Question] Do we have the plan for multiple real devices supported? Since xcodebuild has been supported both real devices and simulators now?

### ISSUE THREAD

**Issue by phoebusliang** [facebook/idb#415]

**Comment by wacban** [comment]
Multiple devices are supported; did you run into any issues when using fbsimctl with multiple devices at the same time?

**Comment by phoebusliang** [comment]
@wacban You mean that we have already support run tests as parallel on multi-physical devices, not simulators?

**Comment by wacban** [comment]
yes, we do support running tests on multiple devices in parallel, feel free to try it out!

yeah the status doesn't work for devieces

**Comment by lawrencelomax** [comment]
If there is an issue with the status appearing as 'Unknown' that sounds like something that is fixable, it's just state formatting. If there are any issues with parallel test execution that you are encountering?

**Comment by Usipov** [comment]
@phoebusliang, did you find out how to launch tests on real devices?

**Comment by zeyadsalloum** [comment]
We just released idb. a new tool that supports devices and simulators We recommend switching over. As part of our renewed commitment to this repo we are cleaning up old issues and will be monitoring and reviewing more closely from now on. if you still think this issue is relevant feel free to reopen

**Comment by Usipov** [comment]
@zeyadsalloum, can you share a link to idb docs (especially to docs about running on real devices)?
Thanks in advance!

**Comment by zeyadsalloum** [comment]
you should find all the docs here www.fbidb.io. feel free to open another issue if you face any difficulties.
